cmd/imagestream.go

```diff
@@ -20,6 +23,9 @@ type ImageStreamCleanupOptions struct {
 	Namespace   string
 	Tag         bool
 	Sorted      string
+	Orphan      bool
+	Duration    string
```

Comment:
`Duration` is very generic and wrong in this use case. How about `OrphanDelta` or something?

Reply:
The duration works not just for the orphans, it's a universal flag

Reply:
Changed the name of the variable from `duration` to `OlderThan`, if you have any other proposals let me know.